certification/internal/operatorsdk/operatorsdk.go

```diff
@@ -1,29 +1,32 @@
-package engine
+package operatorsdk
 
 import (
 	"bytes"
+	"context"
 	"encoding/json"
 	"fmt"
 	"os"
 	"os/exec"
 	"strings"
 
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/artifacts"
-	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/internal/cli"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/runtime"
 	log "github.com/sirupsen/logrus"
 )
 
-func NewOperatorSdkEngine(userProvidedScorecardImage string) *cli.OperatorSdkEngine {
-	var engine cli.OperatorSdkEngine = operatorSdkEngine{scorecardImage: userProvidedScorecardImage}
+func NewOperatorSdk(userProvidedScorecardImage string, cmdContext execContext) *operatorSdk {
```

Comment:
SUPERNIT: Should this just be...
```suggestion
func New(userProvidedScorecardImage string, cmdContext execContext) *operatorSdk {
```

Reply:
Totally agree!